modules/extension/ysld/src/test/java/org/geotools/ysld/parse/YsldParseTest.java

```diff
@@ -1240,13 +1235,13 @@ public void testColourMapEmpty() throws Exception {
         ColorMap map = (ColorMap) symb.getColorMap();
 
         System.out.println(map.getColorMapEntry(0).getColor().evaluate(null));
-        Color colour1 = (Color) map.getColorMapEntry(0).getColor().evaluate(null);
-        Color colour2 = (Color) map.getColorMapEntry(1).getColor().evaluate(null);
-        Color colour3 = (Color) map.getColorMapEntry(2).getColor().evaluate(null);
+        String colour1 = (String) map.getColorMapEntry(0).getColor().evaluate(null);
+        String colour2 = (String) map.getColorMapEntry(1).getColor().evaluate(null);
+        String colour3 = (String) map.getColorMapEntry(2).getColor().evaluate(null);
 
-        assertThat(colour1, is(Color.RED));
-        assertThat(colour2, is(Color.GREEN));
-        assertThat(colour3, is(Color.BLUE));
+        assertThat(colour1, is("#FF0000"));
+        assertThat(colour2, is("#00FF00"));
+        assertThat(colour3, is("#0000FF"));
```

Comment:
These tests should still work as-is. If they don't, you may have broken some other behaviour in YSLD.